src/main/resources/add-ons-to-index.json

```diff
@@ -22,7 +24,9 @@
       }
     },
     {
-      "uid": "org.openmrs.module.accesslogging",
+      "uid": "org.omodopenmrs.module.accesslogging",
+      "moduleId": "1",
```

Comment:
This is not right. The moduleId of this module is `accesslogging`. See for example https://github.com/openmrs/openmrs-module-htmlformentry/blob/3.3.2/omod/src/main/resources/config.xml#L5 . By convention this is always going to be the part of "modulePackage" that comes after the "org.openmrs.module.", and you can write a script to update this file reflecting that convention.